docs/containers/bridge-to-kubernetes.md

```diff
@@ -29,7 +27,7 @@ This guide uses the [Bike Sharing sample application][bike-sharing-github] to de
 * An Azure subscription. If you don't have an Azure subscription, you can create a [free account](https://azure.microsoft.com/free).
```

Comment:
Would it be possible to mention (maybe as a banner?) that while this quickstart is for AKS, Bridge to Kubernetes is also provided as a Preview for other Kubernetes clusters.
Looking at the prerequisites it looks a bit like they are for Bridge and not just for the quickstart, so having this message might help.